pkg/server/container_execsync.go

```diff
@@ -55,6 +56,22 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 		return nil, fmt.Errorf("container %q is in %s state", id, criContainerStateToString(meta.State()))
 	}
 
+	// Get exec process spec.
+	cntrResp, err := c.containerService.Get(ctx, &containers.GetContainerRequest{ID: id})
+	if err != nil {
+		return nil, fmt.Errorf("failed to get container %q from containerd: %v", id, err)
```

Comment:
iow what's the difference between failing to get the container when pulling an image vs failing to get the container here?